pkg/vere/disk.c

```diff
@@ -22,6 +23,12 @@ struct _cd_save {
   struct _u3_disk* log_u;
 };
 
+/* DISK FORMAT
+ */
+
+#define U3D_VER1   1
+#define U3D_VERLAT U3L_VER1
+
```

Comment:
these should be defined in version.h. You're already importing it anyway.

Reply:
Arguably, the pier/log format is a higher-level concern, and shouldn't be part of the core noun package. Doesn't matter match, and will be easy to change later.

Reply:
version.h is in /noun "just cuz." I don't consider it to be part of any package.

@matthew-levan U3D_VERLAT evaluates to U3L_VER1 which isn't defined. It should be U3D_VER1 instead